webpack.config.js

```diff
@@ -140,6 +140,11 @@ const resolve = {
 	modules: [ projectPath( '.' ), 'node_modules' ],
 };
 
+// Get the app version from the google-site-kit.php file - optional chaining operator not supported here
+const googleSiteKitFile = fs.readFileSync( path.resolve( __dirname, 'google-site-kit.php' ), 'utf8' );
+const googleSiteKitVersion = googleSiteKitFile.match( /(?<='GOOGLESITEKIT_VERSION',\s+')\d+.\d+.\d+(?=')/ig );
+const GOOGLESITEKIT_VERSION = googleSiteKitVersion ? googleSiteKitVersion[ 0 ] + '_' : '';
```

Comment:
This results in the version having an underscore appended, which is only necessary in the `STORAGE_KEY_PREFIX` – it shouldn't be part of the version itself. I realize this was part of the IB but perhaps that detail was overlooked in the review. Let's keep the version to only include that which was parsed from the plugin file and add the underscore to the storage prefixes.

Also, we already have a function for parsing the plugin version in `bin/release` so it would be nice if we could share that in both places, since they both run in a node context but there isn't really a good way to do that right now. I'll open an issue to address this in the future.

Reply:
No problem, updated.